lib/internal/Magento/Framework/Search/Adapter/Mysql/Query/Builder/Match.php

```diff
@@ -73,6 +76,9 @@ public function __construct(
         $this->fulltextHelper = $fulltextHelper;
         $this->fulltextSearchMode = $fulltextSearchMode;
         $this->preprocessors = $preprocessors;
+        $this->_ConfigInterface = $ConfigInterface;
+        $this->minQueryLength = (int)$this->_ConfigInterface->getValue(
```

Comment:
Hi @ravi-chandra3197 I wouldn't reference catalog search configuration path in the framework, can you refactor the fix to move the config reference out of the framework?